Tests/Sources/Resources/LocalTweaks_test_override.json

```diff
@@ -1,10 +1,5 @@
 {
   "ui_customization": {
-    "display_blue_view": {
-      "Title": "Display Blue View",
-      "Group": "UI Customization",
-      "Value": true
-    },
```

Comment:
Not sure why this one is gone?

Reply:
I added this only to test the changes, committed by mistake in first Commit.